src/commands/edge_app.rs

```diff
@@ -415,9 +415,12 @@ impl EdgeAppCommand {
         manifest: &EdgeAppManifest,
     ) -> Result<(), CommandError> {
         const SLEEP_TIME: u64 = 2;
-        const MAX_WAIT_TIME: u64 = 20; // 20 seconds
+        const MAX_WAIT_TIME: u64 = 1000; // 1000 seconds - it could take a while for assets to process
```

Comment:
Maybe something like this would be better to explicitly check timeouts 
```
loop {
        if start_time.elapsed().as_secs() > MAX_WAIT_TIME {
            return Err(CommandError::AssetProcessingTimeout);
        }```